instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/TracerProperties.java

```diff
@@ -16,18 +16,26 @@
 
 package io.opentelemetry.instrumentation.spring.autoconfigure;
 
+import javax.validation.constraints.DecimalMax;
+import javax.validation.constraints.DecimalMin;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
  * Configuration for OpenTelemetry Tracer
  *
- * <p>Configures LoggingExporter and sets default tracer name
+ * <p>Get Tracer Name {@link getName()}
```

Comment:
Here and elsewhere: You don't need usually need to provide a list of the methods of the class in its documentation. IS there a special convention for beans?